docs/models/fg.md

```
@@ -1,253 +1,607 @@
+ #ohxzltvyqq .gt_footnote { margin: 0px; font-size: 90%; padding-top: 4px; padding-bottom: 4px; padding-left: 5px; padding-right: 5px; }
+ #ohxzltvyqq .gt_sourcenotes { color: #333333; background-color: #FFFFFF; border-bottom-style: none; border-bottom-width: 2px; border-bottom-color: #D3D3D3; border-left-style: none; border-left-width: 2px; border-left-color: #D3D3D3; border-right-style: none; border-right-width: 2px; border-right-color: #D3D3D3; }
+ #ohxzltvyqq .gt_sourcenote { font-size: 90%; padding-top: 4px; padding-bottom: 4px; padding-left: 5px; padding-right: 5px; text-align: left; }
+ #ohxzltvyqq .gt_footnote_marks { font-size: 75%; vertical-align: 0.4em; position: initial; }
+ #ohxzltvyqq .gt_asterisk { font-size: 100%; vertical-align: 0; }
+ &#10;</style>
+
+| Field goals over expected — 2025 (min 15 attempts) |  |  |  |  |  |  |
+|----|----|----|----|----|----|----|
+| made minus the model-expected makes on the same attempts; kicker identity is deliberately NOT a model feature, so this residual is the kicker |  |  |  |  |  |  |
+|  | Kicker | Team | Att | Made | xMade | FGOE |
+| <img src="https://static.www.nfl.com/image/upload/%7BformatInstructions%7D/league/l4rrfcuczjau7q02h2iz" height="40" /> | E.Pineiro | SF | 32 | 31.0 | 26.0 | 5.0 |
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)